.github/workflows/ci.yml

```diff
@@ -231,16 +240,18 @@ jobs:
         uses: ./.github/actions/unit-test-setup
 
       # Not sure why, but needs to re-build otherwise there's some failures starting up
+      # Smoke tests should cover scenarios that could be broken by changes in other modules that quarkus
       - name: Run Quarkus integration Tests
         run: |
           declare -A PARAMS
-          PARAMS["sanity-check-zip"]="-Dtest=StartCommandDistTest,StartDevCommandDistTest,BuildAndStartDistTest,ImportAtStartupDistTest"
+          PARAMS["win"]="-Dtest=StartCommandDistTest,StartDevCommandDistTest,BuildAndStartDistTest,ImportAtStartupDistTest"
           PARAMS["zip"]=""
           PARAMS["container"]="-Dkc.quarkus.tests.dist=docker"
           PARAMS["storage"]="-Ptest-database -Dtest=PostgreSQLDistTest,MariaDBDistTest#testSuccessful,MySQLDistTest#testSuccessful,DatabaseOptionsDistTest,JPAStoreDistTest,HotRodStoreDistTest,MixedStoreDistTest,TransactionConfigurationDistTest,ExternalInfinispanTest"
+          PARAMS["smoke"]="-Dtest=ClusterConfigDistTest,CustomJpaEntityProviderDistTest,ExportDistTest,FeaturesDistTest,ImportAtStartupDistTest,ImportDistTest,JaxRsDistTest,TruststoreDistTest"
```

Comment:
I tried to identify tests that would potentially break by changes in the main (non-Quarkus related) codebase. These tests also run relatively fast, around 12 minutes in GHA.